cldf/forms.csv

```diff
@@ -213338,7 +213338,7 @@ ID,Local_ID,Language_ID,Parameter_ID,Value,Form,Segments,Comment,Source,Cognacy,
 243-11-430-1,,243,11-430,nyia,nyia,,"Also means 'gold, silver' 09.640, 09.650.",perez1987,,,latintrans,nɨia,phonemic
 243-11-510-1,,243,11-510,zipquazagasqua,zipquazagasqua,,"Means 'to make rich, become rich'.",perez1987,,,latintrans,c̷̣-ipkua-c̷̣-aɣa-skua,phonemic
 243-11-540-1,,243,11-540,ataban mague,ataban mague,,,perez1987,,,latintrans,a-taba-n maɣe,phonemic
-243-11-620-1,,243,11-620,--btyusuca,--btyusuca,,Object precedes verb.,perez1987,,,latintrans,--β-tɨu-suka,phonemic
+243-11-620-1,,243,11-620,-btyusuca,-btyusuca,,Object precedes verb.,perez1987,,,latintrans,-β-tɨu-suka,phonemic
```

Comment:
What is this normalization based on? Above, a leading `-`is removed, here it's left in?

Reply:
it Key's original data CD-ROM a em-dash was loaded as '--' which was nothing else as a normal hyphen